api-extractor/src/definitions/ApiPackage.ts

```diff
@@ -74,6 +74,22 @@ export default class ApiPackage extends ApiItemContainer {
     }
   }
 
+  /**
+   * Find a member in this package by name and return it if found.
+   * 
+   * @param memberName - the name of the member ApiItem
+   */
+  public getMemberItem(memberName: string): ApiItem {
+    let matchedApiItem: ApiItem = undefined;
+    this.memberItems.forEach(apiItem => {
```

Comment:
This is an O(n) algorithm. If we have M things that are doing lookups, it turns into O(n\*m).  It would be better to use a Map, which reduces it to O(1\*m).

The difference seems inconsequential, but I've seen several cases where code that worked fine with 100 items became very slow with 1000 items, because of the big jump from 100\*100=1,000 --> 1000\*1000=1,000,000 loops.